.github/workflows/ci.yml

```diff
@@ -45,7 +44,7 @@ jobs:
         run: |
           mkdir build
           cd build
-          cmake .. -DBUILD_TESTS=OFF -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1s/ -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl@1.1/1.1.1s/lib/
+          cmake .. -DBUILD_TESTS=OFF -DOPENSSL_ROOT_DIR=/usr/local/Cellar/openssl@1.1/1.1.1t/ -DOPENSSL_LIBRARIES=/usr/local/Cellar/openssl@1.1/1.1.1t/lib/
```

Comment:
we will have to make changes in the code every time a new version of openSSL is released. Can we update the code to download the latest version of OpenSSL instead?

Reply:
If I install openssl@1.1 using brew on my mac, it is downloading and installing `1.1.1q` version instead of `1.1.1t`. 
```
$ brew install openssl@1.1
Warning: openssl@1.1 1.1.1q is already installed and up-to-date.
To reinstall 1.1.1q, run:
  brew reinstall openssl@1.1

ls /usr/local/Cellar/openssl@1.1/
1.1.1g	1.1.1q
```

Reply:
Discussed with team and decided to have a followup change for mac os.